packages/blocks/src/api/serializer.js

```diff
@@ -238,7 +238,7 @@ export function getCommentAttributes( blockType, attributes ) {
 			}
 
 			// Ignore all local attributes
-			if ( attributeSchema.__experimentalRole === 'local' ) {
+			if ( attributeSchema.role === 'local' ) {
```

Comment:
Should it also map when interacting the experimental role’s name? Although the `local` role is new so it might not need special handling.

Reply:
Yeh I've just noticed a few of these instances. Thanks for helping me find these 👍